src/canmatrix/tests/test_canmatrix.py

```diff
@@ -731,14 +731,14 @@ def test_define_for_float():
 def test_canid_parse_values():
     can_id = canmatrix.canmatrix.CanId(canmatrix.ArbitrationId(id=0x01ABCD02, extended=True))
     assert can_id.source == 0x02
-    assert can_id.destination == 0x01
```

Comment:
Of coure I wrote the tests to match the current code, not to match the J1939 specification :-(